### PR TITLE
AES override feature

### PR DESCRIPTION
**Pull request by fireice-uk**
Add a manual override for AES cpuid.

### CODE REVIEW

File: config.txt [hunk 1]
```diff
@@ -77,6 +77,16 @@ null,
...
 
 /*
+ * Manual hardware AES override
+ *
+ * Some VMs don't report AES capability correctly. You can set this value to true to make the miner use hardware
+ * AES or to false to make it use software AES.
+ * 
+ * WARNING: setting this to true on a CPU that doesn't support hardware AES will crash the miner.
+ */
+"aes_override" : null,
```
Comment (psychocrypt) [inline config.txt]: IMO false is better as default than `null`

Comment (fireice-uk) [inline config.txt]: it is null because the override works both ways - true will always enable, false will always disable

Comment (psychocrypt) [inline config.txt]: That's nice, could you add all tree states to the docu?

Comment (fireice-uk) [inline config.txt]: `You can set this value to true to make the miner use hardware AES or to false to make it use software AES.` Any idea if that can be further clarified?

Comment (psychocrypt) [inline config.txt]: Maybe: `You can set this value to true to enforce hardware AES or to false to force disable AES or null to let the miner decide if AES is used.`

**Pull request merged**